pkg/pillar/cmd/baseosmgr/baseosmgr.go

```diff
@@ -38,54 +38,15 @@ const (
 // Set from Makefile
 var Version = "No version specified"
 
-type baseOsMgrContext struct {
```

Comment:
I wonder if it would be easier to review the 3rd commit if it was split into two commits; one commit which moves the definition of *Context and random global variables to a separate context.go file (but see Milan's comment about the name), and separate one which changes how the parsing is done.

Reply:
I decided to not rename/split logic in the PR. Will do separately.